src/views/ipv/page-multiple-doc-check.njk

```diff
@@ -1,19 +1,32 @@
 {% extends "shared/base.njk" %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
-{% set pageTitleName = 'pages.pageMultipleDocCheck.title' | translate %}
-{% set googleTagManagerPageId = "pageMultipleDocCheck" %}
+{% if context == 'f2f' %}
+  {% set pageText = "pages.pageF2FMultipleDockCheck" %}
+  {% set googleTagManagerPageId = "pageF2FMultipleDockCheck" %}
+{% else %}}
+  {% set pageText = "pages.pageMultipleDocCheck" %}
+  {% set googleTagManagerPageId = "pageMultipleDocCheck" %}
+{% endif %}}
+
+{% set pageTitleName = [ pageText , '.title' ] | join | translate %}
 
 {% block content %}
-  <h1 class="govuk-heading-l" id="header" data-page="{{googleTagManagerPageId}}">{{'pages.pageMultipleDocCheck.header' | translate }}</h1>
-  <p class="govuk-body">{{'pages.pageMultipleDocCheck.content.paragraph1' | translate | safe }}</p>
+  <h1 class="govuk-heading-l" id="header" data-page="{{googleTagManagerPageId}}">{{[ pageText , '.header' ] | join | translate }}</h1>
+  <p class="govuk-body">{{[ pageText , '.content.paragraph1' ] | join | translate | safe }}</p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>{{'pages.pageMultipleDocCheck.content.bullet1' | translate | safe }}</li>
-    <li>{{'pages.pageMultipleDocCheck.content.bullet2' | translate | safe }}</li>
+    <li>{{[ pageText , '.content.bullet1' ] | join | translate | safe }}</li>
+    <li>{{[ pageText , '.content.bullet2' ] | join | translate | safe }}</li>
   </ul>
-  <p class="govuk-body">{{'pages.pageMultipleDocCheck.content.paragraph2' | translate | safe }}</p>
+  <p class="govuk-body">{{[ pageText , '.content.paragraph2' ] | join | translate | safe }}</p>
+
+  {% if (isWelsh) and (context == 'f2f') %}
```

Comment:
Rather than hardcoding the context value, can we pass in a constant somehow? (Or define a flag eg. `isF2fEnabled` to pass in)